packages/components/src/components/Forms/FormText.tsx

```diff
@@ -0,0 +1,167 @@
+import { Text } from 'react-native'
+import React, { FC } from 'react'
+
+import { TextWithA11y } from '../../types/common'
+import { getA11yText, getDisplayText } from '../../utils'
+import { useTheme } from '../../utils'
+import { useTranslation } from 'react-i18next'
+
+export type FormTextProps = {
+  text: string | TextWithA11y
+}
+
+/**
+ * Fonts (TODO: replace with typography tokens)
+ */
+const fontRegular = 'SourceSansPro-Regular'
+const fontBold = 'SourceSansPro-Bold'
+
+const fontHeader = {
+  fontFamily: fontRegular,
+  fontSize: 20,
+  lineHeight: 30,
+}
+
+const fontHint = {
+  fontFamily: fontRegular,
+  fontSize: 16,
+  lineHeight: 22,
+}
+
+const fontError = {
+  fontFamily: fontBold,
+  fontSize: 16,
+  lineHeight: 22,
+}
+
+export const fontLabel = {
+  fontSize: 20,
+  lineHeight: 30,
+}
+
+const fontDescription = {
+  fontFamily: fontRegular,
+  fontSize: 16,
+  lineHeight: 22,
+}
+
+/**
```

Comment:
Could potentially split these all into separate files instead

Reply:
One file seems fine to me since most of what's in here besides this will be deleted with Typography tokens and it isn't a huge list that feels unwieldy.